### PR TITLE
[IMP] add relation to sales team

### PR DESCRIPTION
**Pull request by damdam-s**

### CODE REVIEW

File: sale_automatic_workflow/i18n/fr.po [hunk 1]
```diff
@@ -78,9 +82,15 @@ msgid "Force Invoice Date"
...
+#: field:automatic.workflow.job,id:0
+#: field:sale.workflow.process,id:0
 msgid "ID"
-msgstr ""
+msgstr "ID"
+
+#. module: sale_automatic_workflow
+#: help:sale.workflow.process,warning:0
+msgid "If set, display the message when a user selects the process on a sale order"
+msgstr "If set, display the message when a user selects the process on a sale order"
```
Comment (guewen) [inline sale_automatic_workflow/i18n/fr.po]: Could you remove the untranslated terms from the translated values?

**Comment by pedrobaeza** [comment]
This should be done in a separate auto-installed module _sale_automatic_workflow_sale_team_.

**Comment by gurneyalex** [comment]
:+1:

**Comment by damdam-s** [comment]
@pedrobaeza sales-team module is already in `depends` list of `sale` module. I added this dependency only to be more explicit. But in anycase, it doesn't force the user to use salesteam. Maybe I missed something but this dependency already exists ?

**Comment by pedrobaeza** [comment]
Sorry, you're right. You select via security group if show sales team.

File: sale_automatic_workflow/sale_workflow_process_view.xml [hunk 2]
```diff
@@ -43,6 +44,7 @@
                     <field name="picking_policy"/>
                     <field name="order_policy"/>
                     <field name="invoice_quantity"/>
+                        <field name="section_id"/>
```
Comment (pedrobaeza) [inline sale_automatic_workflow/sale_workflow_process_view.xml]: Put same indentation level

**Comment by pedrobaeza** [comment]
Minor nitpicking, but :+1:

**Comment by damdam-s** [comment]
@pedrobaeza thanks. indentation done

**Comment by pedrobaeza** [comment]
Can you also please squash the commits in one to not pollute too much repository with a lot of commits?

File: sale_automatic_workflow/sale_workflow_process_view.xml [hunk 3]
```diff
@@ -21,6 +21,7 @@
                         <field name="picking_policy"/>
                         <field name="order_policy"/>
                         <field name="invoice_quantity"/>
+                        <field name="section_id"/>
```
Comment (guewen) [inline sale_automatic_workflow/sale_workflow_process_view.xml]: Could you add `groups="base.group_multi_salesteams"`?

File: sale_automatic_workflow/sale_workflow_process_view.xml [hunk 4]
```diff
@@ -43,6 +44,7 @@
                     <field name="picking_policy"/>
                     <field name="order_policy"/>
                     <field name="invoice_quantity"/>
+                    <field name="section_id"/>
```
Comment (guewen) [inline sale_automatic_workflow/sale_workflow_process_view.xml]: Could you add `groups="base.group_multi_salesteams"`?

**Comment by damdam-s** [comment]
fixed

**Comment by guewen** [comment]
Thanks! :+1:

**Comment by pedrobaeza** [comment]
Merging.

**Pull request merged**